### PR TITLE
Silky's patches

### PR DESCRIPTION
**Pull request by silkyjohnson1**
Remove 2fa radio button; fix toggles on receive tab; update logo.

### CODE REVIEW

**Comment by ReverseControl** [comment]
Looks good. After get some independent testing I will add it to the repo. Just need to confirm and that sending/receiving works as intended and that the button is gone.

**Comment by silkyjohnson1** [comment]
removed lightning/auto updates

**Comment by silkyjohnson1** [comment]
removed a bunch of lightning stuff.

**Comment by silkyjohnson1** [comment]
block explorers fixed.

**Comment by ReverseControl** [comment]
Great. This will take care of two items on the board. I will test later today before merging. Looks good on a first read.

Syncing time improvement would be great to have if you have time to look at it.

File: electrum/util.py [hunk 1]
```diff
@@ -889,68 +889,20 @@ def age(
             return _("in over {} years").format(round(distance_in_minutes / 525600))
 
 mainnet_block_explorers = {
-    '3xpl.com': ('https://3xpl.com/bitcoin/',
+    '3xpl.com': ('https://explorer.fact0rn.io/',
```
Comment (silkyjohnson1) [inline electrum/util.py:892]: line should be

'fact0rn.io': ('https://explorer.fact0rn.io/',

**Comment by silkyjohnson1** [comment]
> Great. This will take care of two items on the board. I will test later today before merging. Looks good on a first read.
> 
> Syncing time improvement would be great to have if you have time to look at it.

can you give me some more information about this? i don't see anything on this mechanism as it relates to network.py. if you have something i can research/documentation i can look at it.

**Comment by ReverseControl** [comment]
I updated the project board with the details about improving syncing time using the checkpoints feature.

**Comment by silkyjohnson1** [comment]
checkpoints generated from console and added to json. network.py updated to enable checkpoints.

**Pull request closed without merging**